internal/test/client/client.gen.go

```diff
@@ -24,6 +24,9 @@ type SchemaObject struct {
 	Role      string `json:"role"`
 }
 
+// openIdContextKey is the context key for OpenId security scheme
+type openIdContextKey string
```

Comment:
would we get any benefits from

```suggestion
type openIdContextKey = string
```

I feel this would reduce the limited impact of a breaking change, but not sure if it's worth it for any other reason.

Reply:
Yes, that approach would avoid a breaking change.
However, by using a type alias, it ends up being the same as using the string type, so it doesn't change the situation.

```go
package main

import (
	"context"
	"fmt"
)

type keyType1 = string // type alias
type keyType2 = string

const key1 keyType1 = "KEY"
const key2 keyType2 = "KEY"

func main() {
	ctx := context.Background()
	ctx = context.WithValue(ctx, key1, "VALUE")

	// key1 conflicts with key2
	fmt.Println(ctx.Value(key1)) // => VALUE
	fmt.Println(ctx.Value(key2)) // => VALUE (want: nil)
}
```